estela-api/config/settings/base.py

```diff
@@ -68,8 +68,8 @@
 ALLOWED_HOSTS = env("DJANGO_ALLOWED_HOSTS").split(",")
 
 
-DJANGO_EXTERNAL_APPS = env("DJANGO_EXTERNAL_APPS").split(",")
-EXTERNAL_MIDDLEWARES = env("EXTERNAL_MIDDLEWARES").split(",")
+DJANGO_EXTERNAL_APPS = [x for x in env("DJANGO_EXTERNAL_APPS").split(",") if x]
+EXTERNAL_MIDDLEWARES = [x for x in env("EXTERNAL_MIDDLEWARES").split(",") if x]
 
```

Comment:
Could you add these variable into the core/views.py file, https://github.com/bitmakerla/estela/blob/30599ef1ac3c7473af1605049ab90f602bfc5d3b/estela-api/core/views.py#L18-L26

Something like:
```python
"DJANGO_EXTERNAL_APPS": ",".join(settings.DJANGO_EXTERNAL_APPS),
"EXTERNAL_MIDDLEWARES": ",".join(settings.EXTERNAL_MIDDLEWARES)
```
without these deploy cannot work.

Reply:
Done